modules/applications/pages/ui-builder/bonita-ui-builder.adoc

```diff
@@ -4,10 +4,15 @@
 
 {description}
 
+[IMPORTANT]
```

Comment:
```suggestion
[NOTE]
```

Reply:
`NOTE` is not really visible to a user. This is why I choose `IMPORTANT`.

Reply:
For me this message does not justify the Important look, which is very visible.